utils_linux.go

```diff
@@ -240,7 +241,15 @@ func (r *runner) run(config *specs.Process) (int, error) {
 			return -1, err
 		}
 	}
-	if r.detach {
+	if !r.create {
+		if err := process.Signal(syscall.SIGCONT); err != nil {
+			r.terminate(process)
+			r.destroy()
+			tty.Close()
+			return -1, err
+		}
+	}
+	if r.detach || r.create {
```

Comment:
On Tue, May 17, 2016 at 11:32:28AM -0700, W. Trevor King wrote:
> But somebody needs to stay around to collect the container exit
> code, tear down cgroups, de-register the state, etc., right?

Rephrased in Go, this is mostly “But somebody needs to call
container.Destroy()”, and that's your separate ‘delete’ call.  The
only thing I'm missing is a way to collect the container exit code.